core-guides/working-with-git.md

```diff
@@ -0,0 +1,72 @@
+---
+title: Working with Git
```

Comment:
I am pretty sure there are other guides online on this e.g. help.github.com. My intuition here would be having something short here and largely referencing other guides would be best - we run the risk of going out of date and this is not the core thing we are doing (e.g. comments on Ubuntu vs Windows users).